### PR TITLE
update logdepthbuf shader

### PR DESCRIPTION
**Pull request by 06wj**
When using ```USE_LOGDEPTHBUF_EXT```, the uniform ```logDepthBufFC ``` is only used in the fragment shader, otherwise it is only used in the vertex shader.

### CODE REVIEW

File: src/renderers/shaders/ShaderChunk/logdepthbuf_pars_vertex.glsl [hunk 1]
```diff
@@ -4,8 +4,10 @@
 
 		varying float vFragDepth;
 
+	#else
+	
+		uniform float logDepthBufFC;
+
```
Comment (WestLangley) [inline src/renderers/shaders/ShaderChunk/logdepthbuf_pars_vertex.glsl]: Clean up - remove leading tabs. Two occurrences.

File: src/renderers/shaders/ShaderChunk/logdepthbuf_pars_vertex.glsl [hunk 2]
```diff
@@ -4,8 +4,10 @@
...
 
+	#else
+	
+		uniform float logDepthBufFC;
+	
 	#endif
 
-	uniform float logDepthBufFC;
-
 #endif
```
Comment (WestLangley) [inline src/renderers/shaders/ShaderChunk/logdepthbuf_pars_vertex.glsl]: Clean up - add new line at end of file.

**Comment by WestLangley** [comment]
Nice find. Thanks!

@mrdoob This looks good to me.

File: src/renderers/shaders/ShaderChunk/logdepthbuf_pars_fragment.glsl [hunk 3]
```diff
@@ -1,11 +1,6 @@
...
+	varying float vFragDepth;
 
-	#ifdef USE_LOGDEPTHBUF_EXT
-
-		varying float vFragDepth;
-
-	#endif
-
-#endif
+#endif
```
Comment (WestLangley) [inline src/renderers/shaders/ShaderChunk/logdepthbuf_pars_fragment.glsl]: Please: add new line at end of file.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**